src/envo/envo-edit.owl

```diff
@@ -31615,76 +31633,133 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001406> ObjectSomeValuesFrom(<
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001410> "altostratus cloud"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001410> <http://purl.obolibrary.org/obo/ENVO_01001428>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001410> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_03520001>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001410> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0002124>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001410> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/BFO_0000027> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_01000846> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002351> <http://purl.obolibrary.org/obo/ENVO_01000833>)))))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001410> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_00002006> <http://purl.obolibrary.org/obo/ENVO_01000277>)))
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_01001411> (nimbostratus cloud)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001411> "A cloud which is 1) grey, often dark grey in colour, 2) organised as a layer, 3) diffused by more or less continuously falling rain or snow, which in most cases reaches the ground, and 3) thick enough throughout to blot out the appearance of the local star.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001411> "A cloud which 1) constitutes a layer, 2) is composed of water droplets (sometimes supercooled) and raindrops, or snow crystals and snowflakes, or of a mixture of these liquid and solid particles, 3) is located in the middle level of the atmosphere (middle étage)."@en)
```

Comment:
"constitutes" - try "is present as a layer" or "occurs in the form of a layer"

Reply:
Again, thanks for spotting this. As above, I've changed it to say "occurs in the form of" instead of "constitutes"